docs/source/tutorial/documentation.rst

```diff
@@ -135,7 +135,7 @@ Here we've followed a particular style guide; Sphinx uses `Google's documentatio
 <https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html>`_ by default to parse
 your docstrings, more on this later! NumPy's `documentation guidelines
 <https://numpydoc.readthedocs.io/en/latest/>`_ are also a great reference for more information
-about what and how to document your code. There are other style guides you might prefer.
+about what and how to document in your code. There are other style guides you might prefer.
```

Comment:
I think you're technically correct on this one, though I do only ever hear "document your code".

Reply:
To me it's the 'what' that changes it.  "How to document your code", but "what to document _in_ your code" - hence I leaned for the "in" in the combined "what and how" construct.